### DISCUSSION TITLE
0.7.2 sectors list wrong param count

### DISCUSSION THREAD

**Discussion by creaz79** [filecoin-project/lotus#4034] [General Q&A] [answered]
Problem starts from version 0.7.2

1. Run 'lotus-miner sectors list'
2. ERROR: RPC error (-32602): wrong param count (method 'Filecoin.StateMinerSectors'): 3 != 4

**Expected behavior**
List of sectors

**Version (run `lotus version`):**
lotus version 0.7.2+git.eaf34249

lotus version 0.7.1+git.4121063c.dirty  output in attach
[lotus_miner_err.txt](https://github.com/filecoin-project/lotus/files/5285751/lotus_miner_err.txt)

**Comment by jennijuju** [comment, marked as answer]
Did you update both your miner and daemon? I just tested it out and it works for me.

(I will convert this to discussion for now, and we can reopen the ticket if it still doesnt work for you after 0.8.0 upgrade 🤗

**Reply by creaz79** [reply]
This problem gone with 0.8.0. Thank you.
